dataplane/standalone/apigen/apigen.go

```diff
@@ -330,20 +586,91 @@ func generate() error {
 			if strings.Contains(tf.TypeName, "PORT_ALL") || strings.Contains(tf.TypeName, "ALL_NEIGHBOR") {
 				tf.UseCommonAPI = false
 			}
-			data.Funcs = append(data.Funcs, tf)
+			ccData.Funcs = append(ccData.Funcs, tf)
```

Comment:
I'm ok with this since this is prototype code, but if this block is self-contained functionality then having a function would be nice.

Reply:
agreed, added todo